### PR TITLE
HackKnight Challenge Flow

### PR DESCRIPTION
**Pull request by tandebnath**

### CODE REVIEW

File: components/HackKnightChallenge/KnightChallenge.module.scss [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        row-gap: 0.5rem;
+    }
+
+    .block {
+        display: flex;
+        flex-flow: column nowrap;
+        row-gap: 0.75rem;
+    }
+
+    .colouredText {
```
Comment (lasyaneti) [inline components/HackKnightChallenge/KnightChallenge.module.scss:69]: can you give the colored text (inputs/urls) a mode mono/code-link font

File: components/HackKnightChallenge/KnightChallenge.module.scss [hunk 2]
```diff
@@ -0,0 +1,153 @@
+.container {
+    display: flex;
+    flex-flow: column nowrap;
+    align-items: center;
+    row-gap: 2rem;
+}
+
+.box {
+    background-color: #a61e00;
```
Comment (lasyaneti) [inline components/HackKnightChallenge/KnightChallenge.module.scss]: the contrast is a bit harsh imo, i asked design to give us a new background color to make the challenge more readable

File: components/HackKnightChallenge/KnightChallenge.tsx [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+                                Feel free to email us if you have issues with
+                                the coding challenge! Please direct your email
+                                to{" "}
+                                <a href="mailto:lasya.neti@hackillinois.org">
+                                    <u>lasya.neti@hackillinois.org</u>
+                                </a>{" "}
+                                and{" "}
+                                <a href="mailto:aydan.pirani@hackillinois.org">
+                                    <u>aydan.pirani@hackillinois.org</u>
+                                </a>
```
Comment (lasyaneti) [inline components/HackKnightChallenge/KnightChallenge.tsx]: replace emails with `challenge@hackillinois.org`

File: components/HackKnightChallenge/KnightChallenge.tsx [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+                                            max_goodness
+                                        </span>{" "}
+                                    in the request body as shown below.
+                                    </p>
+                                    <p
+                                        className={`${styles.colouredText} ${styles.marginTop} ${styles.text}`}
+                                    >{`POST https://artemis.hackillinois.org/challenge`}</p>
+                                    <p
+                                        className={`${styles.colouredText} ${styles.marginTop} ${styles.text}`}
+                                    >{`{"max_goodness": token_here}`}</p>
```
Comment (lasyaneti) [inline components/HackKnightChallenge/KnightChallenge.tsx]: this line should say `{"max_goodness": integer_value_here}` (not token - my bad)

**Comment by nrking0** [comment]
Merged into Sarena's branch, so I am closing this PR.

**Pull request closed without merging**